package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "middy",
-  "version": "0.2.4",
+  "version": "0.3.0",
```

Comment:
let's keep this `0.2.5` for now. I now it's not strictly semantic versioning (as we are adding new features here and not just doing bug fixes), but I'd like to keep the version count "small" before an official release.